### DISCUSSION TITLE
AttributeError: 'NoneType' object has no attribute 'split'

### DISCUSSION THREAD

**Discussion by parmenx** [swar/Swar-Chia-Plot-Manager#244] [General]
[config.txt](https://github.com/swar/Swar-Chia-Plot-Manager/files/6463832/config.txt)
When I enter the python manager.py view command;

```
(venv) C:\Users\win\Documents\Swar-Chia-Plot-Manager-main>python manager.py view
Traceback (most recent call last):
  File "C:\Users\win\Documents\Swar-Chia-Plot-Manager-main\manager.py", line 36, in <module>
    view()
  File "C:\Users\win\Documents\Swar-Chia-Plot-Manager-main\plotmanager\library\utilities\commands.py", line 91, in view
    drive = directory.split('\\')[0]
AttributeError: 'NoneType' object has no attribute 'split'
```

When I look from the task manager, it is plotting. but I don't have a monitoring screen. 

My yaml file is attached.

**Comment by djnapone** [comment]
I'm having the same issue.  The jobs are running but I just can't see the display.  I was messing around with disk space clieaning and python updates when this occurs.
